tools_iuc.yaml.lock

```diff
@@ -1144,7 +1144,6 @@ tools:
   revisions:
   - 38c8bb402872
   - d958d5a0d1e8
-  - e0a23ab32d7f
```

Comment:
@gmauro you need to manually remove this tool as ephemeris will not deinstall tools atm.

Reply:
yes, I have done it